docs/standard/linq/toc.yml

```diff
@@ -302,21 +302,21 @@
           href: program-nodes.md
         - name: Mixed declarative code-imperative code bugs
           href: mixed-declarative-code-imperative-code-bugs.md
-        - name: "How to stream XML fragments with access to header information"
+        - name: How to stream XML fragments with access to header information
           href: stream-xml-fragments-access-header-information.md
-        - name: "How to perform streaming transform of large XML documents"
-          href: perform-streaming-transform-large-xml-documents.md
-        - name: "How to read and write an encoded document"
+        - name: How to do streaming transform of large XML documents
+          href: do-streaming-transform-large-xml-documents.md
```

Comment:
even though acrolinx doesn't like perform, I actually prefer the previous title and file name

Reply:
I did the do -> perform changes